go.mod

```
@@ -1,11 +1,11 @@
 module github.com/couchbase/sync_gateway
 
-go 1.22.0
+go 1.23.0
```

```suggestion
go 1.24.0
```

or not? I don't actually know

---

I could bump this requirement, but we _can_ build on 1.23.0 so I left it here because it matches what we did on main. My thought was we don't need to bump this until a dependency requires it? But it doesn't matter either way.

I think there's something to putting `toolchain 1.24.2` in the files to enforce using that version of go, which is sort of a belt and suspenders approach to also remembering to update the Jenkinsfile, github actions, product-config.json - since it will enforce using that version. It would also force using that version locally.

I'm inclined to leave this as is for release/3.2.4 though and puzzle the right thing to do on main when it is less likely to break builds.

---

That's fine by me. Is this PR now ready for review/merge?